Doc/library/tkinter.font.rst

```diff
@@ -93,4 +102,10 @@ The different font weights and slants are:
 
 .. function:: nametofont(name)
 
-   Return a :class:`Font` representation of a tk named font.
\ No newline at end of file
+   Return a :class:`Font` representation of a tk named font.
+
+References
+----------
+
+.. [1] Tk 8.6 font man page
+   https://www.tcl.tk/man/tcl8.6/TkCmd/font.htm#M5
```

Comment:
This will be very useful to many who wouldn't know where to find this otherwise!